zig/.gitignore

```
@@ -0,0 +1,6 @@
+LICENSE
+README.md
+doc
+lib
+zig
+zig.exe
```

Do we want to be explicit about what's in the `zig` tarball? If not, we could do

```suggestion
*
!download.bat
!download.sh
```

Even if we do, we might want to still do this and then keep the manual `mv`'ing to not have 2 places to update.

---

Also, no trailing newline.